### ISSUE TITLE
USER ISSUE: 

### ISSUE THREAD

**Issue by Selene-Salvatore** [StrangeLoopGames/EcoIssues#14494]
**Version:** 0.8.3.2 beta

**Steps to Reproduce:**
Does not load Worlds. I always get failed connection

**Expected behavior:**
Loaded my Worlds. 

**Actual behavior:**
The following error message popped up:
```
<size=150%>Connection Failed</size>

Failed to connect - connection timed out.
```

**Do you have mods installed? Does issue happen when no mods are installed?:**
The following error message popped up:
```
<size=150%>Connection Failed</size>

Failed to connect - connection timed out.
```

**Comment by Selene-Salvatore** [comment]
I have installed it new did not work. And I do not have any mods installed

**Comment by SlayksWood** [comment]
@Selene-Salvatore 
1. Please provide log file.
Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
2. Do you have Firewall or Antivirus enabled?
3. Which kind of internet connection are you using?
4. Please provide your PC specification (include Operating System).
5. Did you meet this issue after updating to the "latest build" or you had it before?
6. Can you try to connect to official servers (Sea Otter and Giant Tortoise) and provide log files?
Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by Selene-Salvatore** [comment]
[Player.log.copy.txt](https://github.com/StrangeLoopGames/EcoIssues/files/4108264/Player.log.copy.txt)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/4108266/Player-prev.log)

Here are the logs

**Comment by Selene-Salvatore** [comment]
I only have the Win 10 Firewall enabled

**Comment by Selene-Salvatore** [comment]
I am using W-Lan

**Comment by Selene-Salvatore** [comment]
Where do I find the complete Pc specifications? Sorry I don't know much about Computers.

**Comment by Selene-Salvatore** [comment]
Yes before the last update I did not have this issue. It just started with the Update

**Comment by Selene-Salvatore** [comment]
6. the instructions link is the same as 1. 
I don't know how to do Nr. 6

**Comment by SlayksWood** [comment]
@Selene-Salvatore Did you try to connect to offical server?

**Comment by Selene-Salvatore** [comment]
Yes i tried and it works 
I can connect to the official Servers but not to my Worlds

**Comment by D3nnis3n** [comment]
Please upload the server logs after you tried to start one of your worlds.
The same instructions as posted above do apply.

**Comment by Selene-Salvatore** [comment]
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/4111904/Player-prev.log)
[Player.log.copy.txt](https://github.com/StrangeLoopGames/EcoIssues/files/4111905/Player.log.copy.txt)

**Comment by Selene-Salvatore** [comment]
I tried loading it again got the error and hopefully these are the right logs. I could only find playler.log.copy instead of player.log

**Comment by D3nnis3n** [comment]
No, you need to upload the SERVER logs, the article describes where they are. They are in the game folder of Eco.

**Comment by Selene-Salvatore** [comment]
I did that but there is nothing in there :/

**Comment by Selene-Salvatore** [comment]
I did just what you said but the Logs are empty

**Comment by Selene-Salvatore** [comment]
[log_200125123754.log](https://github.com/StrangeLoopGames/EcoIssues/files/4112452/log_200125123754.log)

Hey i think i found it

**Comment by Selene-Salvatore** [comment]
Hey I managed to fix my Problem.
I just deleted the whole Game File and then reinstalled the Game. Something must have been wrong in the mods file. I do not know what since I did not mod anything in Eco but it does not matter now. I made a Backup of my Saves bevore deleting everything and just put the Backup back in there.  And it works now :) Thank you for your help without you I would have never tried deleting the files.
